SavingAccount.rb

```diff
@@ -0,0 +1,30 @@
+module Bank
+
+  class SavingAccount < Account
+    attr_reader :balance
+    # set my deafualt balance to 100
+    def initialize(id, balance, opendate)
+      super
+      @withdraw_fee = 2
```

Comment:
I like that you used a variable to store the information associated with the fee